README.md

```diff
@@ -43,59 +44,94 @@ Contains the 3D models of the external components and their supports
-    - **big_plate_support.SLDASM (Alternative with a laser cut technology)**: Assembly of four 1/8" 2D cut plate
-      - **big_support_long_side_0P125.SLDPRT**: Long side of the box
-      - **big_support_small_side_0P125.SLDPRT**: Small side of the box
-      - TODO: Need a small bracket to connect the box to the screw holes of the pioneer
+  - **big_plate_support.SLDPRT**: Big stand that can be used to support the front of the plate inside the robot. This part is optional because the four small supports are able to maintain the plate stable.
 
 - **internal_electronics_plate.SLDASM**: Assembly of the circuits components and the support plate.
 
   ![alt text](plate_components.PNG)
 
-  - **bornier.SLDPRT**: Screw terminals.
-  - **diode_schottky.SLDPRT**: Schottky diode.
+  - **bornier.SLDPRT**: Screw terminals block.
   - **REG_313080006.SLDPRT**: 313080006 5V regulator
-  - **relay.SLDPRT**: Relay
+  - **Relays**
+    - **relay.SLDPRT**: Relay
+    - **relay_bracket.SLDPRT**: Bracket to hold the relay at a 90 degrees angle. This help with the electrical harness.
+    - **relay_assembly.SLDASM**: Assembly of the relay and its bracket
   - **roboclaw_2X30A.SLDASM**: Motor controller CAD. Got it on : [RoboClaw 2x30A Motor Controller (basicmicro.com)](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html)
 
   
 
-- **Side pannel**
+- **Computer's side pannel**
 
   ![alt text](side_pannel.PNG)
 
-  - **computer.SLDPRT**: Box that represents the footprint of  Raspberry Pi 4 and a Jetson Nano ( sizes of their longest side with height of a Jetson)
-  - **side_pannel.SLDPRT**: Model of a panel that can be installed on the left-side slot of the pioneer to support a min-plate. In our case, we used is to support the computer.
+  - **computer.SLDPRT**: Box that represents the footprint of  Raspberry Pi 4 and a Jetson Nano ( sizes of their longest side with height of a Jetson). The CAD of the Raspberry Pi 4 is very heavy to load so this box is used as a lighter representation.
+
+  - **side_pannel.SLDPRT**: Model of a panel that can be installed on the left-side slot of the pioneer to support a small plate. In our case, we used is to support the computer.
+
   - **computer_support.SLDPRT**: Plate that can be screwed on the *side_pannel.SLDPRT* . It has the bolt pattern to install a Rasberry Pi 4 or a Jeston Nano.
 
+  - **computer_support_pi.SLDPRT**: Plate that can be screwed on the *side_pannel.SLDPRT* . It has the bolt pattern to install a Rasberry Pi 4. Was designed to have a more compact plate the selected computer is the Raspberry Pi 4. 
+
+  - **supported_pi.SLDASM**: Assembly of the side pannel and the Raspberry Pi 4 support plate.
+
+  - **fan_support.SLDPRT**: Support for the 5V fan that comes with the [LABIST Raspberry Pi 4 case]([LABISTS Case for Raspberry Pi 4](https://labists.com/products/raspberry-pi-4-case-kit)). Screwed with the screws that came with the kit.
```

Comment:
Screwed with the screws that came with the kit. -> Use the screws provided with the kit.